### PR TITLE
11.0 mig project_stage_closed

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa

### CODE REVIEW

File: project_stage_closed/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
```
Comment (angelmoya) [inline project_stage_closed/__init__.py]: You can remove copyright on __init__.py files

**Comment by rafaelbn** [comment]
cc @Tecnativa

**Review by AaronHForgeFlow — COMMENTED** [review]
I'm not sure about the purpose of this module, the flag 'closed' in tasks is not existing in Odoo code now, am I wrong?

**Comment by rafaelbn** [comment]
@aheficent  this module add a checkbox in project.task.type to indicate which Stages means that the task is closed or not.

From v8 @sbidoul description: https://github.com/OCA/project/pull/142#issue-53736661

> Odoo provides a 'closed' flag on project task stages, but this flag
is in the sale_service module, which in turns pulls a lot of dependencies
such as sale, accounting, and procurement. In many circumstances,
it is desirable to have such a flag to indicate a task is closed,
without needing so many dependencies.

> This module provides the flag in a way that is compatible with sale_service
but depending only on the project module.

**Comment by sbidoul** [comment]
You may also be interested in [project_task_stage_allow_timesheet](https://github.com/OCA/hr-timesheet/tree/11.0/project_task_stage_allow_timesheet)

**Review by rafaelbn — APPROVED** [review]
Tested functionally :+1:  Please @ernestotejeda review comments in order to merge this one Thanks!

File: project_stage_closed/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Project Task Stage Closed",
+    'summary': "Make the Closed flag on Task Stages available without "
+               "installing sale_service""",
+    'author': 'ACSONE SA/NV,'
+              'Tecnativa,'
+              'Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```
Comment (MeetKD) [inline project_stage_closed/__manifest__.py]: Can you please change website key `https://github.com/OCA/project`

**Review by MeetKD — CHANGES_REQUESTED** [review]

**Comment by AaronHForgeFlow** [comment]
@rafaelbn Thank you for the explanation. Let's move on with this.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on runbot

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**